src/app/portlets/content-types/create/content-types-create.component.spec.ts

```diff
@@ -65,12 +65,12 @@ describe('ContentTypesCreateComponent', () => {
 
     beforeEach(async(() => {
         const messageServiceMock = new MockMessageService({
-            'Content': 'Content',
-            'File': 'File',
-            'Form': 'Form',
-            'Page': 'Page',
-            'Persona': 'Persona',
-            'Widget': 'Widget'
+            'contenttypes.content': 'Content',
+            'contenttypes.file': 'File',
+            'contenttypes.form': 'Form',
+            'contenttypes.page': 'Page',
+            'contenttypes.persona': 'Persona',
+            'contenttypes.widget': 'Widget'
```

Comment:
I think is better something like:

contenttypes.content.label

in this way we can use the prefix 'contenttypes.content' for all the message linked to 'content'

Reply:
Changes were added in the latest commit, thanks!